### PR TITLE
HPCC-28280 Revise and Rename the More Helm and Yaml section

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>
No new content, just revising what was already there. 

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful Unit/Build tests: http://10.224.20.18/view/Docs-gp/job/DocBuild-01-GPRepo

### CODE REVIEW

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 1]
```diff
@@ -1236,89 +1192,50 @@ global:
...
-        selectively set in a users values file, and the other options will be
-        retained.</para>
-      </sect3>
+      previous sections, as well as the Kubernetes and Helm
+      documentation.</para>
 
       <sect3 id="merging_AND_Overrides">
         <title>Merging and Overriding</title>
 
         <para>Having multiple yaml files, such as one for logging, another for
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1218]: consistency: YAML is used elsewhere

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 2]
```diff
@@ -1236,89 +1192,50 @@ global:
...
-        ensuring any non-changing areas are left untouched. The rule here to
-        keep in mind where multiple yaml files are applied, the later ones
-        will always overwrite the values in the earlier ones. They are merged
-        in in sequence.</para>
+        storage, yet another for secrets and so forth, these files can all be
+        under version control. They can be versioned, checked in, etc. and
+        have the benefit of only defining/changing the specific area required,
+        while ensuring any non-changing areas are left untouched.</para>
+
+        <para>The rule here to keep in mind where multiple yaml files are
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1224]: consistency: YAML is uses elsewhere

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Couple of minor comments inline

**Review by JamesDeFabia — APPROVED** [review]
Good from my POV

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 3]
```diff
@@ -1068,76 +1068,76 @@
...
-      a data serialization language often used as a format for configuration
-      files. The construct that makes up the bulk of a yaml file is the
-      key-value pair, sometimes referred to as a hash or a dictionary. The
-      key-value pair construct consists of a key that points to some value(s).
-      The values could be strings, numbers, booleans, integers, arrays, or
-      dictionaries, and lists. These values are defined by the schema.</para>
+      <para>The <emphasis>values.yaml</emphasis> file is a YAML file which is
+      a format frequently used for configuration files. The construct that
+      makes up the bulk of a YAML file is the key-value pair, sometimes
+      referred to as a hash or a dictionary. The key-value pair construct
```
Comment (jakesmith) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1093]: trivial/not new: if you're going to mention "hash", it should really say "hash table", though I'd be inclined not to mention it at all and only say "sometimes referred to as a dictionary" ( "hash table" is more commonly used to describe the implementation )

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 4]
```diff
@@ -1149,78 +1149,34 @@
     - key: "kubernetes.azure.com/scalesetpriority"
 </programlisting></para>
 
-        <para>A key is denoted using a minus sign, which is an entry item in
-        the list, which itself is a dictionary with nested attributes. Then
-        the next minus sign (at that same indentation level) is the next entry
-        in that list.</para>
+        <para>A key is denoted using the hyphen, which is an entry item in the
```
Comment (jakesmith) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1152]: minor/not new: I think the use of the word "key" here is misleading, there is no key field of a list entry.. there is only an order of entries, the hyphen signifies the start of the next entry in the list.
It's an unfortunate example that the list is a dictionary with the a key field called "key" in the above example..
A list doesn't have to be a list of dictionaries, e.g.:
```
mylist: [ 1, 2, 3 ]
```
which is equivalent to:
```
mylist:
- 1
- 2
- 3
```

**Review by jakesmith — APPROVED** [review]
@g-pan - a couple of comments, but they're minor and not new, so approving (but you might want to address them in this PR).

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 5]
```diff
@@ -1236,89 +1194,51 @@ global:
...
 
       <sect3 id="merging_AND_Overrides">
         <title>Merging and Overriding</title>
 
-        <para>Having multiple yaml files, such as one for logging, another for
-        storage, yet another for secrets and so forth, the files can be in
-        version control. They can be versioned, checked in, etc. and have the
+        <para>Having multiple YAML files, such as one for logging, another for
+        storage, yet another for secrets and so forth, allows granular
+        configuration.. These configuration files can all be under version
```
[the remark this answers is not in the document]
Comment (g-pan) [reply docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml]: good catch

**Review by jakesmith — COMMENTED** [review]
One rewording suggestion.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 6]
```diff
@@ -1149,78 +1149,36 @@
...
 
-        <para>A key is denoted using a minus sign, which is an entry item in
-        the list, which itself is a dictionary with nested attributes. Then
-        the next minus sign (at that same indentation level) is the next entry
-        in that list.</para>
+        <para>The list entry here is denoted using the hyphen, which is an
+        entry item in the list, which itself is a dictionary with nested
+        attributes. Then the next hyphen (at that same indentation level) is
+        the next entry in that list. A list can be a dictionary but it doesn't
+        have to be a list of dictionaries, which may in turn also have list
```
[the remark this answers is not in the document]
Comment (g-pan) [reply docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1156]: Thank you Jake, that is a very helpful suggestion to clarify that statement.

**Comment by jakesmith** [comment]
@g-pan - looks good

**Review by JamesDeFabia — APPROVED** [review]
Looks good

**Pull request merged**